src/xunit.analyzers/AssertEqualStringValueShouldNotUseGenericType.cs

```diff
@@ -0,0 +1,37 @@
+﻿using System.Collections.Generic;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Xunit.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    public class AssertEqualStringValueShouldNotUseGenericType : AssertUsageAnalyzerBase
+    {
+        private static readonly HashSet<string> EqualMethods = new HashSet<string>(new[] { "Equal" });
```

Comment:
The original issues are just idea notes I left myself, frequently incomplete. In this case if there is no behavioral difference between `Equal<string>' and `StrictEqual<string>` then both should be fixed to `Equal(string, string)`